test/simulator/main/org/apache/cassandra/simulator/paxos/PairOfSequencesAccordSimulation.java

```diff
@@ -149,7 +149,7 @@ public PairOfSequencesAccordSimulation(SimulatedSystems simulated,
     @Override
     protected String createTableStmt()
     {
-        return "CREATE TABLE " + KEYSPACE + ".tbl (pk int, count int, seq text, PRIMARY KEY (pk))";
+        return "CREATE TABLE " + KEYSPACE + ".tbl (pk int, count int, seq text, PRIMARY KEY (pk)) WITH transactional_mode = 'full'";
```

Comment:
"should" this use `transactionMode` since you are trying to pass that around now?

Reply:
fair, I should pass this through yes